app/Resources/views/line/bundle_card.html.twig

```diff
@@ -1,47 +1,64 @@
-<div class="bundle-card-line{% if extra.page is defined %} page-{{ extra.page }}{% endif %}">
-    <div class="bundle-card-line-data">
-        <div class="col-sm-8 col-md-9 col-lg-10 bundle-card-line-content">
-            <h2>
-                <a href="https://packagist.org/packages/{{ ez_field_value(content, 'bundle_id') }}" title="See on Packagist.org">{{ ez_render_field(content, 'bundle_id') }}</a>
-            </h2>
-            <div class="bundle-card-line-description">
-                {{ ez_render_field(content, 'description') }}
+<div class="bundle-card-line-container{% if extra.page is defined %} page-{{ extra.page }}{% endif %}">
+    <div class="bundle-card-line">
+        <div class="bundle-card-line-data">
+            <div class="bundle-card-line-information">
+                <h1>
+                    {{ ez_render_field(content, 'packagist_url') }}
```

Comment:
It should not be `h1` because a web page "should" only have one `h1`. So for all the bundle cards, please change to `<h2>`